huxley/core/admin/delegate.py

```diff
@@ -29,6 +30,25 @@ def roster(self, request):
 
         return roster
 
+    def load(self, request):
+        '''Loads new Assignments.'''
+        delegates = request.FILES
+        reader = csv.reader(delegates['csv'])
+
+        assignments = {}
+        for assignment in Assignments.objects.all():
+            assignments[committee.name + country.name + school.name] = assignment
```

Comment:
Where are `committee`, `country`, and `school` defined? Did you even test this code?